src/Illuminate/Foundation/Console/StorageLinkCommand.php

```diff
@@ -61,4 +64,16 @@ protected function links()
         return $this->laravel['config']['filesystems.links'] ??
                [public_path('storage') => storage_path('app/public')];
     }
+
+    /**
+     * Checks that the provided path is a symlink and force option is turned on.
+     *
+     * @param  string  $link
+     * @param  bool  $force
+     * @return bool
+     */
+    protected function removableSymlink(string $link, bool $force): bool
```

Comment:
Boolean methods often are prefixed with `is` or `has`:

```suggestion
    protected function isRemovableSymlink(string $link, bool $force): bool
```